test/__snapshots__/TimePicker.spec.js.snap

```diff
@@ -4,57 +4,13 @@ exports[`<TimePicker /> accepts external \`id\` value 1`] = `
 <TextInput
   id="ID"
   inputClassName="timepicker"
-  options={
```

Comment:
how come these values have disappeared from snapshots?

Reply:
Actually, this is a good sign. :stuck_out_tongue_closed_eyes: 
`options` should never be in `snapshots` otherwise it means that the rendered component on the `DOM` would be like

```html
<input class="timepicker" type="text" id="timepicker3" options="[object Object]" value="">
```

which is not correct.

As a matter of fact, here is how the component is now rendered:

```html
<input class="timepicker" type="text" id="timepicker4" value="">
```

Reply:
ping @alextrastero :smile: